### ISSUE TITLE
[Bug]: Stopped working out of nowhere, impossible to open file

### ISSUE THREAD

**Issue by 0verEngineer** [advplyr/audiobookshelf#2500]
### Describe the issue

I listened to an Audiobook and paused it, after an hour or so i tried to continue but it did not start.
I even recreated the docker container.
The files are readable

Container logs:
```
audiobookshelf_1  | Config /config /metadata
audiobookshelf_1  | [2024-01-06 16:55:33.638] INFO: === Starting Server ===
audiobookshelf_1  | [2024-01-06 16:55:33.677] INFO: [Server] Init v2.7.1
audiobookshelf_1  | [2024-01-06 16:55:33.691] INFO: [Database] Initializing db at "/config/absdatabase.sqlite"
audiobookshelf_1  | [2024-01-06 16:55:33.747] INFO: [Database] Db connection was successful
audiobookshelf_1  | [2024-01-06 16:55:33.892] INFO: [Database] Db initialized with models: user, library, libraryFolder, book, podcast, podcastEpisode, libraryItem, mediaProgress, series, bookSeries, author, bookAuthor, collection, collectionBook, playlist, playlistMediaItem, device, playbackSession, feed, feedEpisode, setting
audiobookshelf_1  | [2024-01-06 16:55:33.943] INFO: [BackupManager] 0 Backups Found
audiobookshelf_1  | [2024-01-06 16:55:33.943] INFO: [BackupManager] Auto Backups are disabled
audiobookshelf_1  | [2024-01-06 16:55:33.945] INFO: [LogManager] Init current daily log filename: 2024-01-06.txt
audiobookshelf_1  | [2024-01-06 16:55:33.961] INFO: [Watcher] Initializing watcher for "Audiobooks".
audiobookshelf_1  | Warning: connect.session() MemoryStore is not
audiobookshelf_1  | designed for a production environment, as it will leak
audiobookshelf_1  | memory, and will not scale past a single process.
audiobookshelf_1  | [2024-01-06 16:55:33.983] INFO: Listening on port :80
audiobookshelf_1  | [2024-01-06 16:55:35.330] INFO: [Watcher] "Audiobooks" Ready
audiobookshelf_1  | [2024-01-06 16:55:43.715] INFO: [SocketAuthority] Socket Connected KTHIrwnVvF5xziJRAAAB
audiobookshelf_1  | Error: ENOENT: no such file or directory, stat '/audiobooks/J.K. Rowling/Harry Potter/4_Harry_Potter_und_der_Feuerkelch/CD02/02-Zurück zum Fuchsbau.mp3'
audiobookshelf_1  | [2024-01-06 16:55:49.001] INFO: [PlaybackSessionManager] startSession: Closing open session "4_Harry_Potter_und_der_Feuerkelch" for user "julian" (Device: Windows 10 / Firefox)
audiobookshelf_1  | [2024-01-06 16:55:49.030] INFO: [STREAM] START STREAM - Num Segments: 13677
audiobookshelf_1  | [2024-01-06 16:55:49.045] INFO: [STREAM] Starting Stream at startTime 1:12:44.0 (User startTime 1:13:14.0) and Segment #727
audiobookshelf_1  | [2024-01-06 16:55:49.212] INFO: [INFO] FFMPEG transcoding started with command: ffmpeg -seek_timestamp 1 -f concat -safe 0 -ss 269s -noaccurate_seek -i /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/files.txt -y -loglevel error -map 0:a -c:a copy -f hls -copyts -avoid_negative_ts make_non_negative -max_delay 5000000 -max_muxing_queue_size 2048 -hls_time 6 -hls_segment_type mpegts -start_number 727 -hls_playlist_type vod -hls_list_size 0 -hls_allow_cache 0 -hls_segment_filename /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-%d.ts /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/final-output.m3u8
audiobookshelf_1  | [2024-01-06 16:55:49.213] INFO:
audiobookshelf_1  | [2024-01-06 16:55:49.314] INFO: [concat @ 0x7f999953e740] Impossible to open '/audiobooks/J.K. Rowling/Harry Potter/4_Harry_Potter_und_der_Feuerkelch/CD02/02-Zurück zum Fuchsbau.mp3'
audiobookshelf_1  | [2024-01-06 16:55:49.315] INFO: Error demuxing input file 0: No such file or directory
audiobookshelf_1  | [2024-01-06 16:55:49.315] INFO: /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/files.txt: No such file or directory
audiobookshelf_1  | [2024-01-06 16:55:49.320] INFO:
audiobookshelf_1  | [2024-01-06 16:55:49.321] INFO: [FFMPEG] Transcoding ended
audiobookshelf_1  | [2024-01-06 16:55:49.321] INFO: [STREAM] 0250e3e8-44af-4d36-a1cf-358a98e34e39 notifying client that stream is ready
audiobookshelf_1  | [2024-01-06 16:55:50.253] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:50.495] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:50.689] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:50.881] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:51.084] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:51.282] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:55:51.479] WARN: File path does not exist /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:05.489] INFO: [SocketAuthority] Socket KTHIrwnVvF5xziJRAAAB disconnected from client "julian" after 21774ms (Reason: transport close)
audiobookshelf_1  | [2024-01-06 16:56:06.268] INFO: [SocketAuthority] Socket Connected h6OWWM7FBfbxpzrWAAAD
audiobookshelf_1  | [2024-01-06 16:56:08.880] INFO: [PlaybackSessionManager] startSession: Closing open session "4_Harry_Potter_und_der_Feuerkelch" for user "julian" (Device: Windows 10 / Firefox)
audiobookshelf_1  | [2024-01-06 16:56:08.881] INFO: Closing Stream 0250e3e8-44af-4d36-a1cf-358a98e34e39
audiobookshelf_1  | [2024-01-06 16:56:08.884] INFO: Deleted session data /metadata/streams/0250e3e8-44af-4d36-a1cf-358a98e34e39
audiobookshelf_1  | Error: ENOENT: no such file or directory, stat '/audiobooks/J.K. Rowling/Harry Potter/4_Harry_Potter_und_der_Feuerkelch/CD02/02-Zurück zum Fuchsbau.mp3'
audiobookshelf_1  | [2024-01-06 16:56:09.121] INFO: [PlaybackSessionManager] startSession: Closing open session "4_Harry_Potter_und_der_Feuerkelch" for user "julian" (Device: Windows 10 / Firefox)
audiobookshelf_1  | [2024-01-06 16:56:09.150] INFO: [STREAM] START STREAM - Num Segments: 13677
audiobookshelf_1  | [2024-01-06 16:56:09.170] INFO: [STREAM] Starting Stream at startTime 1:12:44.0 (User startTime 1:13:14.0) and Segment #727
audiobookshelf_1  | [2024-01-06 16:56:09.180] INFO: [INFO] FFMPEG transcoding started with command: ffmpeg -seek_timestamp 1 -f concat -safe 0 -ss 269s -noaccurate_seek -i /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/files.txt -y -loglevel error -map 0:a -c:a copy -f hls -copyts -avoid_negative_ts make_non_negative -max_delay 5000000 -max_muxing_queue_size 2048 -hls_time 6 -hls_segment_type mpegts -start_number 727 -hls_playlist_type vod -hls_list_size 0 -hls_allow_cache 0 -hls_segment_filename /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-%d.ts /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/final-output.m3u8
audiobookshelf_1  | [2024-01-06 16:56:09.181] INFO:
audiobookshelf_1  | [2024-01-06 16:56:09.264] INFO: [concat @ 0x7f0edd977740] Impossible to open '/audiobooks/J.K. Rowling/Harry Potter/4_Harry_Potter_und_der_Feuerkelch/CD02/02-Zurück zum Fuchsbau.mp3'
audiobookshelf_1  | [2024-01-06 16:56:09.265] INFO: Error demuxing input file 0: No such file or directory
audiobookshelf_1  | [2024-01-06 16:56:09.265] INFO: /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/files.txt: No such file or directory
audiobookshelf_1  | [2024-01-06 16:56:09.269] INFO:
audiobookshelf_1  | [2024-01-06 16:56:09.270] INFO: [FFMPEG] Transcoding ended
audiobookshelf_1  | [2024-01-06 16:56:09.271] INFO: [STREAM] f7678699-366a-4aba-975b-0ab07e0c3752 notifying client that stream is ready
audiobookshelf_1  | [2024-01-06 16:56:11.028] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:11.263] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:11.455] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:11.665] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:11.883] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:12.105] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
audiobookshelf_1  | [2024-01-06 16:56:12.289] WARN: File path does not exist /metadata/streams/f7678699-366a-4aba-975b-0ab07e0c3752/output-733.ts (HlsRouter.js:75)
```

### Steps to reproduce the issue

I have not done anything besides using the pause and play button ^^

### Audiobookshelf version

v2.7.1

### How are you running audiobookshelf?

Docker

**Comment by 0verEngineer** [comment]
It works again after i rescanned the library with the root user, the interesting thing is 18 of my audiobooks are now reporting "Item directory is missing!" but the files are there and the permissions are the same as for the working audiobooks...

**Comment by advplyr** [comment]
I'm not sure. Did you resolve the issue?

**Comment by 0verEngineer** [comment]
I will check it tomorrow again :)

**Comment by AZgeekster** [comment]
Version 2.7.1
Similar/same errors occurring. It occurs randomly when I am playing audiobooks using the android app that seem to play for a bit of time, but then stop playing and give me an error about unable to play stream, and these errors are present in audiobookshelf. The audiobook then fails to play until I restart the abs docker container.

Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:44:50.043

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:44:52.043

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:44:54.051

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:44:56.048

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:44:58.047

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'
2024-01-12 23:45:00.053

ERROR
Failed checking files Error: ENOENT: no such file or directory, scandir '/metadata/streams/53ca1e06-5261-4dd7-9b55-8016e8469cb3'

**Comment by 0verEngineer** [comment]
I checked the problem with the missing audiobooks after re-scan today and they were just rediscovered under a new name, most of the time the name is better (real book name) as before. I did not have the file errors again after the initial ones.

**Comment by itsREV** [comment]
I ran into this issue on  v2.34.0. Turned out it was because i had mounted my SMB shared in my VM and it had unmounted itself at some point. Remounting the share in my VM fixed the issue.

Log excerpt: 

`Error opening input files: No such file or directory
&#34;
[2026-05-14 19:15:11.686] INFO: Closing Stream 66077fae-b793-43cd-a24c-c909023d5477
[2026-05-14 19:15:11.688] INFO: Deleted session data /metadata/streams/66077fae-b793-43cd-a24c-c909023d5477`
